core/field.js

```diff
@@ -992,3 +992,13 @@ Blockly.Field.prototype.getParentInput = function() {
   }
   return parentInput;
 };
+
+/**
+ * Handles the action given.
```

Comment:
Maybe add a little more here describing when this is triggered. ie: only when keyboard nav accessibility mode is on.